apps/desktop-wallet/src/features/ledger/useInitializeAppWithLedgerData.tsx

```diff
@@ -1,4 +1,5 @@
 import { NonSensitiveAddressData } from '@alephium/keyring'
+import { walletUnlockedDesktop } from '@alephium/shared'
```

Comment:
The reason of this naming will be explained below, at the definition of the action.